ERCS/erc-7955.md

```diff
@@ -166,34 +166,36 @@ The `CREATE2_FACTORY_SALT` was chosen as the **first** salt value starting from
 
-Note that this contract returns the address of the created contract padded to 32 bytes. This differs from some existing implementations, but was done to maintain consistency with the 32-byte word size on the EVM (same encoding as `ecrecover` precompile for example). A product of this is that the return data from CREATE2 factory is compatible with the Solidity ABI.
-
-Throughout both the CREATE2 factory contract initialization and runtime code, we use `RETURNDATASIZE (0x3d)` to push `0` onto the stack instead of the dedicated `PUSH0 (0x5f)` opcode. This is done to increase compatibility with chains that support EIP-7702 but not [EIP-3855](./eip-3855.md), while remaining a 1-byte and 2-gas opcode.
+Throughout both the CREATE2 factory contract initialization and runtime code, `RETURNDATASIZE (0x3d)` is used to push `0` onto the stack instead of the dedicated `PUSH0 (0x5f)` opcode. This is done to increase compatibility with chains that support EIP-7702 but not [EIP-3855](./eip-3855.md), while remaining a 1-byte and 2-gas opcode.
 
 The `CREATE2_FACTORY_INIT_CODE` corresponds to the following assembly:
 
 ```
+# SPDX-License-Identifier: CC0-1.0
+
 ### Constructor Code ###
 
-0x0000: PUSH24 0x60203d3d3582360380843d373d34f580601457fd5b3d52f3
+0x0000: PUSH29 0x60203d3d3582360380843d373d34f5806019573d813d933efd5b3d52f3
                         # Stack: [runcode]                      | Push the CREATE2 factory runtime code
-0x0019: RETURNDATASIZE  # Stack: [0; runcode]                   | Push the offset in memory to store the code
-0x001a: MSTORE          # Stack: []                             | The runtime code is now in `memory[8:32]`
-0x001b: PUSH1 25        # Stack: [24]                           | Push the code length
-0x001d: PUSH1 7         # Stack: [8; 24]                        | Push the memory offset of the start of code
-0x001f: RETURN          # Stack: []                             | Return the runtime code
+0x001e: RETURNDATASIZE  # Stack: [0; runcode]                   | Push the offset in memory to store the code
+0x001f: MSTORE          # Stack: []                             | The runtime code is now in `memory[8:32]`
```

Comment:
Is the comment here still correct? It says `memory[8:32]` and below we push `3` as the offset.

Also was it correct before, as we pushed `7`?

Reply:
Also minor request: can we add why it is at 3, so that it is because bytes are left-padded to 32bytes and we use 29, therefore the offset is 3

Reply:
Very nice catch!